evm/utils.js

```diff
@@ -751,6 +751,8 @@ const mainProcessor = async (options, processCommand, save = true, catchErr = fa
         return;
     }
 
+    const chainsSnapshot = JSON.parse(JSON.stringify(config.chains));
```

Comment:
```suggestion
    const chains = JSON.parse(JSON.stringify(config.chains));
```
maybe call it `chains` or `chainsConfig` for consistency?